src/Features/Lsif/GeneratorTest/OutputFormatTests.vb

```diff
@@ -132,15 +247,15 @@ Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
     ""inVs"": [
       4
     ],
-    ""id"": 10,
+    ""id"": 12,
     ""type"": ""edge"",
     ""label"": ""contains""
   },
   {
     ""kind"": ""end"",
     ""scope"": ""project"",
     ""data"": 2,
-    ""id"": 11,
+    ""id"": 13,
```

Comment:
if we added semantic tokens to the end, these woudln't have to update, right?

Reply:
Probably.